build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts

```diff
@@ -89,21 +89,21 @@ const getCodeCoverageSummaryForPackages = (coverageReport: CodeCoverageCompariso
 
 	return `<details><summary><b>${getColorGlyph(coverageReport.branchCoverageDiff + coverageReport.lineCoverageDiff)} ${
 		coverageReport.packagePath
-	}:</b> <br> Line Coverage Change: ${formatDiff(coverageReport.lineCoverageDiff)}  Branch Coverage Change: ${formatDiff(
+	}:</b> <br> Line Coverage Change: ${formatDiff(coverageReport.lineCoverageDiff)}&nbsp;&nbsp;&nbsp;&nbsp;Branch Coverage Change: ${formatDiff(
 		coverageReport.branchCoverageDiff,
 	)}</summary>${metrics}</table></details>`;
 };
 
 const getColorGlyph = (codeCoverageDiff: number): string => {
 	if (codeCoverageDiff === 0) {
-		return "■";
+		return "&rarr;";
```

Comment:
If we go with these HTML-escaped codes, let's rename the function to `getColorGlyphForHtml` to make it clear the glyphs it produces won't work in console output, for example.